README.md

```diff
@@ -79,11 +79,23 @@ This initiates Webpack, enabling the user to visualize the module online. Direct
 
 ### Testing
 
-To run tests: 
+To run tests:
 
 ```sh
 npm run test
 ```
 
+### CRUD Functionality
+
+GET Request -- send requests to '/api/moreplacestostay'. As currently written, the server will respond will all places and you will have to sort on front-end. This will be fixed in later versions of SDC as its untenable.
+
+DELETE Request -- send requests to '/api/moreplacestostay/:id' where id is unique identifier for the location you would like to delete.
+
+POST Request -- send requests to '/api/placestostay/create.' In the body of your post request, include name (as string), price(as integer), imageurl(as string).
+
+PUT request -- send requests to '/api/placestostay/:id' where id is unique identifier for the location you would like to delete. In the body of your post request, include name (as string), price(as integer), imageurl(as string).
+
+
+
```

Comment:
@rogen-code 
I'd maybe consider adding some basic routing examples similar to https://expressjs.com/en/starter/basic-routing.html , just so it's super clear even from just the README how that implementation could look. Sort of like how bullet points are more skimmable than paragraphs, some straight-forward examples could be helpful.